shared-libs/validation/test/validations.js

```diff
@@ -18,15 +20,22 @@ const stubMe = (functionName) => {
 };
 
 describe('validations', () => {
+  let reportGetUuids;
+  let qualifier;
 
   beforeEach(() => {
     db = { medic: { query: () => stubMe('query'), allDocs: () => stubMe('allDocs') } };
+    dataContext = { bind: sinon.stub() };
     config = {};
     translate = sinon.stub().returnsArg(0);
     sinon.stub(logger, 'debug');
     sinon.stub(logger, 'error');
 
-    validation.init({ db, config, translate });
+    reportGetUuids = sinon.stub();
+    dataContext.bind.returns(reportGetUuids);
+    qualifier = sinon.stub(Qualifier, 'byFreetext');
```

Comment:
note: no need to make any changes here, but in the `api` unit tests we have tended to not mock out the `Qualifier` functions.  In my experience, there comes a point of diminishing returns for mocking lower level functions where it takes more effort to mock the behavior than it does to just manage the behavior in the test code.  Personally, I think the qualifier code fits that case where we might as well just use the real implementation code in the tests....